src/gt4py/next/program_processors/runners/dace_fieldview/gtir_to_tasklet.py

```diff
@@ -761,17 +871,22 @@ def visit_FunCall(self, node: gtir.FunCall) -> IteratorExpr | ValueExpr:
                     connector,
                 )
 
-        assert isinstance(node.type, ts.ScalarType)
-        dtype = dace_utils.as_dace_type(node.type)
-
-        return self._get_tasklet_result(dtype, tasklet_node, "result")
+        if isinstance(node.type, itir_ts.ListType):
+            # make_const_list returns `ListType` but we represent it as a scalar value
+            assert isinstance(node.type.element_type, ts.ScalarType)
+            dtype = dace_utils.as_dace_type(node.type.element_type)
+            return self._get_tasklet_result(dtype, tasklet_node, "result", use_array=True)
```

Comment:
I have nothing against having multiple `return` statements.
However, I think in this case it would increase the clarity if there is just one `return`.

Reply:
Agreed.